### PR TITLE
My work

### PR DESCRIPTION
**Pull request by trodelgal**

### CODE REVIEW

File: client/src/components/Main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const newTicketList = await axios.get(`/api/tickets?searchText=${search}`);
+    setTickets(newTicketList.data);
+  }
+
+  return (
+    <>
+      <div id="searchLine">
+        <div id="webTitle">
+          <h1 id="mainTitle">TICKETS MANAGER</h1>
+          <p id="secondTitle">WE HERE TO HELP OUR CLIENTS</p>
```
Comment (AlonBru) [inline client/src/components/Main.js:52]: We are* here to help our clients

File: client/src/components/Ticket.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <Button id="restoreHideTickets" onClick={restoreTickets}>restore</Button>
+      </span>
+    );
+  }
+
+  return (
+    <div className="liveTicketList">
+      <h1 id="listTitle">TICKETS</h1>
+      <div id="results">
+        Showing
```
Comment (AlonBru) [inline client/src/components/Ticket.js:30]: better add a space here between "showing" and the number

File: MYREADME.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# My work
```
Comment (Cyber4sPopo) [inline MYREADME.md:1]: You should replace the original README with this README

File: client/public/site.webmanifest [hunk 4]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (Cyber4sPopo) [inline client/public/site.webmanifest:1]: Did you add this file intentionally?  If so, why isn't it indented?

File: client/src/components/Content.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import './style/Main.css';
+import Button from '@material-ui/core/Button';
+
+function Content({ contentValue }) {
+  const [contentText, setContentText] = useState(contentValue.slice(0, 450));
```
Comment (Cyber4sPopo) [inline client/src/components/Content.js:6]: Its better to trim the text with CSS `ellipsis`:
https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/
👨🏻‍🏫 - prepare 5 min explanation

File: server/app.js [hunk 6]
```diff
@@ -1,5 +1,55 @@
...
+
+app.post('/api/tickets/:ticketId/undone', async (req, res) => {
+  const tickets = await fs.readFile(dataPathFile, 'utf-8');
+  const jsTickets = JSON.parse(tickets);
+  jsTickets.forEach((ticket, i) => {
+    if (ticket.id === req.params.ticketId) {
+      jsTickets[i].done = false;
+    }
+  });
+  await fs.writeFile(dataPathFile, `${JSON.stringify(jsTickets)}`);
```
Comment (Cyber4sPopo) [inline server/app.js:50]: `fs.writeFile` === `fs.writeFileSync`

File: client/src/myTests.test.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+        .query(() => true)
+        .reply(200, mockData);
+    await page.goto('http://localhost:3000/', { waitUntil: 'networkidle0' })
+    const elements = await page.$$('.ticket');
+    expect(elements.length).toBe(mockData.length);
+    expect(getAllTicketsMock.isDone()).toBe(true);
+    const doneButton = await page.$(".doneButton")
+    await doneButton.click();
+    const elementsAfterDone = await page.$('.ticket');
+    expect(elementsAfterDone.length).toBe(undefined);
```
Comment (Cyber4sPopo) [inline client/src/myTests.test.js:45]: `length` should be `0`, not `undefined`

File: server/app.js [hunk 8]
```diff
@@ -1,5 +1,55 @@
...
+  }
+});
+
+// POST- CHANGE THE TICKET DONE PROPERTY TO TRUE BY ID
+
+app.post('/api/tickets/:ticketId/done', async (req, res) => {
+  const tickets = await fs.readFile(dataPathFile, 'utf-8');
+  const jsTickets = JSON.parse(tickets);
+  jsTickets.forEach((ticket, i) => {
+    if (ticket.id === req.params.ticketId) {
```
Comment (Cyber4sPopo) [inline server/app.js:31]: What if `ticketId` is not found? we should return an error to the client

File: client/src/components/Label.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './style/Label.css';
+
+function Labels({ labels }) {
+  // CHECK IF THE TICKET HAS LABELS
+  let realLabels = [];
+  if (labels !== undefined) {
```
Comment (Cyber4sPopo) [inline client/src/components/Label.js:7]: try using **defaultProps**
👨🏻‍🏫 - prepare 5 min explanation

**Pull request merged**